boot/setup.sh

```diff
@@ -220,7 +156,7 @@ echo "Setting up muskie"
 # Sometimes muskies come up before DNS resolvers are in /etc/resolv.conf
 wait_for_resolv_conf
 manta_setup_muskie
-manta_setup_muskie_rsyslogd
+manta_common2_setup_log_rotation 'muskie'
 
 manta_common_setup_end
```

Comment:
Should this be changed to manta_common2_setup_end?

Reply:
yes! good catch wow, i thought I had changed that but apparently i didn't... will update.